docs/checks/internal/checks.yaml

```diff
@@ -670,6 +670,11 @@ checks:
       This configuration is secure, but there is a chance that when a new job is added to the workflow, its job permissions could be
       left undefined because of human error.
 
+      Though a project's score won't be penalized, the check's details will include
+      warnings for all run-level permissions. This compromise makes it clear the
```

Comment:
It's not all run-level permissions though. Just the ones of "interest" that previously led to point deductions. The README explained why they resulted in point deduction, and I still think there's value in that section about why we WARN for them.

https://github.com/ossf/scorecard/blob/ebf967241d5b7d8548fab04399826c17e390be4a/checks/raw/permissions.go#L32-L47

Reply:
Oh, I knew only a few permissions cost points, but thought all permissions got warnings. Fixed, PTAL.